Parent/ParentUnitTests/Conversations/ConversationDetailViewControllerTests.swift

```diff
@@ -42,14 +51,69 @@ class ConversationDetailViewControllerTests: ParentTestCase {
         super.tearDown()
     }
 
-    func testRender() {
+    func testLayout() {
```

Comment:
Are we calling these layout test now instead of render tests?

Reply:
You can have multiple render tests, but I wouldn't think that means naming the method `testRender`, which sounds like a unit test on a `render()` function, which UIKit doesn't have. It's not a big deal though, I can change it back if you prefer the previous naming.